lib/install/elm.rb

```diff
@@ -19,4 +19,7 @@
 run "./bin/yarn add --dev elm-hot-loader elm-webpack-loader"
 run "yarn run elm package install -- --yes"
 
+puts "Updating elm source location"
+gsub_file "elm-package.json", /\"\.\"\n/, %("app/javascript/packs"\n)
```

Comment:
@bkuhlmann I see then in that case, lets just pull in the raw paths from paths.yml - `File.join(Webpacker::Configuration.source, Webpacker::Configuration.paths.fetch(:entry, 'packs'))`